### ISSUE TITLE
Nuclearcraft fusion reactor

### ISSUE THREAD

**Issue by Angelos33** [NillerMedDild/Enigmatica2Expert#134]
#### Issue Description
I am trying to build a fusion reactor to make tritium and neutron fluid. i find some tutorials in the internet but every time my reactor explodes when it hits the 8000 kK temperature mark. Is that a bug or something? I don't know what else to do. i get that its not a modpack issue but i don' know where to report it.
#### Modpack version (Do not use "latest")
Modpack Version 1.30

**Comment by NillerMedDild** [comment]
I haven't received any reports about this, so I suspect it's simply overheating?

**Comment by NillerMedDild** [comment]
Here's a good tutorial: https://www.youtube.com/watch?v=L-0LOtg2Plk&t=15s

**Comment by Angelos33** [comment]
i watched this tutorial and others but i didn't do anything different and it still explodes. it explodes even after i disabled the overheating in the config.

**Comment by Angelos33** [comment]
i tried a lot of things. i re-downloaded the pack and put back my save and then i saw in the installed mods section of the modpack that i could manually update the nuclearcraft mod so i did and this seems to fix the problem of exploding at 8000kK. But i am a bit confused cause now the problem is fixed and the mod version is the NuclearCraft 2.10p. So i don't know with what version the 1.30 version of the modpack comes with and therefor what cause the problem in the first place so i can help if someone has the same issue.

**Comment by NillerMedDild** [comment]
1.30 uses NuclearCraft 2.10p, so I guess you were just on the wrong version?

**Comment by turbodiesel4598** [comment]
Yeh, I'm guessing that's all that happened.

**Comment by dtroitskiy** [comment]
Hey guys! I know this was asked long ago, but I just encountered same problem and it's actually very little info one can find on this topic. In my case problem was in that I initially charged my magnet ring, but then power supply for it depleted and looks like that causes the explosion. I couldn't figure that out for few hours, was totally confused, thought it's some kind of bug. Then I noticed that even after reactor is started, power consumption from my network is still high, so I figured out that only magnets can do this, and consequently I improved my power supply. Just writing this in hope someone else will find it here, desperately seeking answer just like I did now =)
